examples/chain/11_MsgBatchCreateSpotLimitOrders/example.go

```diff
@@ -58,15 +57,14 @@ func main() {
 	}
 
 	ctx := context.Background()
-	marketsAssistant, err := core.NewMarketsAssistantUsingExchangeClient(ctx, exchangeClient)
+	marketsAssistant, err := chainclient.NewMarketsAssistantInitializedFromChain(ctx, exchangeClient)
```

Comment:
`NewMarketsAssistantInitializedFromChain` This function looks complicated to  me, how about `NewMarketsAssistant`?

Reply:
`NewMarketsAssistant` already exists. It is the function that creates the markets assistant by reading the ini files